packages/main/src/plugin/index.ts

```diff
@@ -926,6 +927,17 @@ export class PluginSystem {
     this.ipcHandle('os:getHostname', async (): Promise<string> => {
       return os.hostname();
     });
+    this.ipcHandle('os:getHostFreeDiskSize', async (): Promise<number> => {
+      const i = (await checkDiskSpace(os.homedir()));
```

Comment:
is it time consuming ?

Reply:
in my case not much but i am on a new laptop, maybe this could be a factor ...  btw this can be changed based on what we decide to do on your other question.

Reply:
I was just wondering if we had to cache the value